packages/prerender-data-provider/index.js

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'preact';
+import { createContext } from 'preact-compat';
```

Comment:
just really not happy about requiring compat to use prerender data

Reply:
me too... dont know a better way though.
Thats y extracted it out in a different pckg

Reply:
only reasonable way I see would be to require preact x, we have it in core there

Reply:
Like only supported for preactX?

Reply:
Yes

Reply:
fixed.
added compatibility note in readme